headers/evernode.h

```diff
@@ -423,4 +423,35 @@ const uint8_t evr_currency[20] = GET_TOKEN_CURRENCY(EVR_TOKEN);
         _06_08_ENCODE_DROPS_FEE(fee_ptr, fee);                                                                       \
     }
 
+#define IS_HOST_PRUNEABLE(res, host_addr, cur_moment_type)                                                                                             \
```

Comment:
Let's take the flag as the last param (Since other `IS_` macros has also followed the same convention), And rename the flag to `is_prunable`